### PR TITLE
Announcement for haskell summer of code 2023

### PR DESCRIPTION
**Pull request by aaronallen8455**
Updates the homepage with an announcement for HSoC

### CODE REVIEW

**Comment by aaronallen8455** [comment]
How it looks rendered:
Screenshot 2023-03-20 at 9 46 26 PM

**Comment by idabzo** [comment]
fantastic work @aaronallen8455!!! 
What I would like to suggest:
Separate ourselves from gsoc resources (https://buildyourfuture.withgoogle.com/programs/summer-of-code and https://developers.google.com/open-source/gsoc/help/student-stipends) - can we create our own subpages and copy-paste the content from there? If we mix HSoC with GSoC info materials, people might be confused.

File: content/ideas.html [hunk 1]
```diff
@@ -1,14 +1,8 @@
-<p class="notification">
-  <strong>Update</strong>: Sadly, Haskell was not selected to take part in GSoC
-  2023. On the bright side, we are intending to run an independent <i>Haskell
-  Summer of Code</i> program instead. Details are forthcoming.
-</p>
-
-<h1>GSoC 2023 Ideas</h1>
+<h1>Haskell Summer of Code 2023 Ideas</h1>
```
Comment (david-christiansen) [inline content/ideas.html]:
```suggestion
<h1>Summer of Haskell 2023 Ideas</h1>
```

File: content/ideas.html [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
 
 <p>
   This is a list of ideas for contributors who are considering to apply to
-  Google Summer of Code 2023 for <em>Haskell.org</em>.
+  Haskell Summer of Code 2023 for <em>Haskell.org</em>.
```
Comment (david-christiansen) [inline content/ideas.html]:
```suggestion
  Summer of Haskell 2023.
```

File: content/index.md [hunk 3]
```diff
@@ -6,25 +6,105 @@ title: Home
...
+experienced mentors. This program is open to all newcomers to open source, age
+18 years or older, in most countries.
 
-<img src="/images/gsoc-logo.svg" alt="Google Summer of Code"
-        style="width: 100%;">
+# <img src="/images/hsoc-logo.svg" alt="Haskell Summer of Code" style="width:90px"> Haskell Summer of Code 2023
 
-# Google Summer of Code 2023
+We are pleased to announce that <em>Haskell.Org</em> in collaboration with the
+<em>Haskell Foundation</em> will be hosting the <b>Haskell Summer of Code
```
Comment (david-christiansen) [inline content/index.md]:
```suggestion
<em>Haskell Foundation</em> will be hosting the <b>Summer of Haskell
```

File: content/index.md [hunk 4]
```diff
@@ -6,25 +6,105 @@ title: Home
...
+    </tr>
+  </table>
+</div>
+
+### Sponsors
+
+We'd like to give special thanks to the <em>Haskell Foundation</em> for
+providing the funds for a portion of the students, and to these individuals and
+organizations who have also pledged to sponsor students:
+
```
Comment (gbaz) [inline content/index.md:99]: Add HF Sponsorship

File: content/index.md [hunk 5]
```diff
@@ -6,25 +6,105 @@ title: Home
...
 
-Google Summer of Code 2023
-[has been announced](https://opensource.googleblog.com/2022/11/get-ready-for-google-summer-of-code-2023.html)
+### Participation
 
-We are currently [looking for ideas](/ideas.html) that students/contributors
-could work on during Google Summer of Code 2023.
+We will begin accepting proposals from prospective contributors starting April
+3rd and the deadline for submissions is April 20th. If you would like to apply,
+please send an email to `hsoc [AT] haskell.org` with a Google doc containing
```
Comment (david-christiansen) [inline content/index.md]:
```suggestion
please send an email to `summer [AT] haskell.org` with a Google doc containing
```

File: content/index.md [hunk 6]
```diff
@@ -6,25 +6,105 @@ title: Home
...
+providing the funds for a portion of the students, and to these individuals and
+organizations who have also pledged to sponsor students:
+
+ -  Gershom Bazerman and Edward Kmett
+
+If you or your organization would like to sponsor a student, please [contact us](/contact.html)!
+
+---
+
+<img src="/images/gsoc-logo.svg" alt="Google Summer of Code" style="width: 100%;">
```
Comment (david-christiansen) [inline content/index.md]: s/Google/Haskell/?

File: content/faq.md [hunk 7]
```diff
@@ -4,4 +4,10 @@ title: Frequently Asked Questions
 
 # Faq
 
-Google Summer of Code's FAQ can be found [here](https://developers.google.com/open-source/gsoc/faq)
+Our policies are largely similar to that of Google Summer of Code, please refer
+to their FAQ page found [here](https://developers.google.com/open-source/gsoc/faq).
+
+Some notable exceptions to Googles policy are:
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
Some notable exceptions to Google's policies are:
```

File: content/faq.md [hunk 8]
```diff
@@ -4,4 +4,10 @@ title: Frequently Asked Questions
...
+
+ -  Contributors can apply for more than one project
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
 -  Contributors to Summer of Haskell can apply for up to two projects
```

File: content/faq.md [hunk 9]
```diff
@@ -4,4 +4,10 @@ title: Frequently Asked Questions
...
+ -  We are following a slightly different timeline (see [homepage](./))
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
 -  Summer of Haskell follows a slightly different timeline (see [homepage](./))
```

File: content/index.md [hunk 10]
```diff
@@ -6,27 +6,99 @@ title: Home
 
 The Summer of Haskell is an effort by Haskell.Org to reach out to students and
 encourage them to contribute to the Haskell community with the aid of
-experienced mentors.  This program is open to college students, 18 or older, in
-most countries.
+experienced mentors. This program is open to all newcomers to open source, age
```
Comment (david-christiansen) [inline content/index.md]:
```suggestion
experienced mentors. This program is open to all college students and newcomers to open-source Haskell contributions, age
```

File: content/index.md [hunk 11]
```diff
@@ -6,27 +6,99 @@ title: Home
...
 
-<img src="/images/gsoc-logo.svg" alt="Google Summer of Code"
-        style="width: 100%;">
+# <img src="/images/hsoc-logo.svg" alt="Summer of Haskell" style="width:90px"> Summer of Haskell 2023
 
-# Google Summer of Code 2023
+We are pleased to announce that <em>Haskell.Org</em> in collaboration with the
+<em>Haskell Foundation</em> will be hosting the <b>Summer of Haskell 2023</b>!
+This program will be occurring independently from <i>Google Summer of Code</i>
+but offers the same opportunities and amenities, including mentorship and a
```
Comment (david-christiansen) [inline content/index.md]:
```suggestion
but offers the same opportunities, including mentorship and a
```

File: content/index.md [hunk 12]
```diff
@@ -6,27 +6,99 @@ title: Home
...
+stipend, to would-be Haskell contributors.
 
-Google Summer of Code 2023
-[has been announced](https://opensource.googleblog.com/2022/11/get-ready-for-google-summer-of-code-2023.html)
+### Participation
 
-We are currently [looking for ideas](/ideas.html) that students/contributors
-could work on during Google Summer of Code 2023.
+We will begin accepting proposals from prospective contributors starting April
+3rd and the deadline for submissions is April 20th. If you would like to apply,
```
Comment (david-christiansen) [inline content/index.md]:
```suggestion
3rd, and the deadline for submissions is April 20th. If you would like to apply,
```

File: content/stipend.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
+---
+title: Stipend
+---
+
+# Stipend policy
+
+The Summer of Haskell is independent from Google Summer of Code but our stipend
+policy will match theirs. The guidelines are available [here](https://developers.google.com/open-source/gsoc/help/student-stipends#total_stipend_amount).
```
Comment (david-christiansen) [inline content/stipend.md]:
```suggestion
policy matches theirs. The guidelines are available [here](https://developers.google.com/open-source/gsoc/help/student-stipends#total_stipend_amount).
```

**Pull request merged**